src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const titleCheck = movie.title.toLowerCase();
+    const descriptionCheck = movie.description.toLowerCase();
+
+    const queryCheck = query.trim().toLowerCase();
+
+    return titleCheck.includes(queryCheck)
+      || descriptionCheck.includes(queryCheck);
```

Comment:
Names of variables are confusing at this point. 'titleCheck.includes(queryCheck)' better call it 'preparedTitle' or 'formattedTitle' in all cases.

Reply:
agree